### PR TITLE
Updates based on O feedback

### PR DESCRIPTION
**Pull request by hthillman**

### CODE REVIEW

**Review by yondonfu — APPROVED** [review]

File: docs/video-miners/how-to-guides/l2-migration.md [hunk 1]
```diff
@@ -43,26 +43,22 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
     
 2. **Follow the prompt to begin migration to Arbitrum**
     
-    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amount will not appear until after you enter your Ethereum account address.
-    
-    <img src="/docs-assets/video-miners/how-to-guides/stake-info.png" alt="stake info" width="300" />
-
-
+    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amounts will not be accurate until after you sign the transaction and we autodetect your orchestrator account from the signature.
+    <img src="/docs-assets/video-miners/how-to-guides/begin-migration.png" alt="begin migration" width="300"/>
```
Comment (adamsoffer) [inline docs/video-miners/how-to-guides/l2-migration.md:47]: This prompt will only be visible if they're on Rinkeby. Should we also point out the link in the sidebar?

Screen Shot 2022-01-24 at 11 55 25 PM
?

File: docs/video-miners/how-to-guides/l2-migration.md [hunk 2]
```diff
@@ -43,26 +43,22 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
...
-    <img src="/docs-assets/video-miners/how-to-guides/sign-web.png" alt="sign web" width="300"/>
+    If you prefer to sign using the wallet that you have connected to the explorer, click "Approve Migration" and approve the transaction using your browser extension
 
+    <img src="/docs-assets/video-miners/how-to-guides/sign-web.png" alt="sign web" width="300"/>
     
     3b. **Sign using CLI**
     
-    If you'd rather sign a typed data message through the CLI, connect your wallet to the Explorer with any other account. You'll be asked to enter the public address of the orchestrator you wish to migrate.
-    
+    If you'd rather sign a typed data message through the CLI, select "Sign using CLI" in the explorer interface. You'll be taken to an alternate migration screen, where you will be asked to enter the public address of the orchestrator you wish to migrate
```
Comment (adamsoffer) [inline docs/video-miners/how-to-guides/l2-migration.md:60]: There's no "Sign using CLI" in the explorer. We simply show the signing steps if we detect it's not an orchestrator.

**Review by adamsoffer — CHANGES_REQUESTED** [review]
Left a few comments

File: docs/video-miners/how-to-guides/l2-migration.md [hunk 3]
```diff
@@ -43,26 +43,22 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
     
 2. **Follow the prompt to begin migration to Arbitrum**
     
-    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amount will not appear until after you enter your Ethereum account address.
-    
-    <img src="/docs-assets/video-miners/how-to-guides/stake-info.png" alt="stake info" width="300" />
-
-
+    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amounts will not be accurate until after you sign the transaction and we autodetect your orchestrator account from the signature.
```
Comment (adamsoffer) [inline docs/video-miners/how-to-guides/l2-migration.md:46]: The correct stake amounts will appear after you enter your public ethereum address. Also should this note can probably be moved underneath the "sign using cli" option?
image

File: docs/video-miners/how-to-guides/l2-migration.md [hunk 4]
```diff
@@ -43,26 +43,22 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
     
 2. **Follow the prompt to begin migration to Arbitrum**
```
Comment (adamsoffer) [inline docs/video-miners/how-to-guides/l2-migration.md:44]: Maybe change this to "Navigate to the L2 Migration Tool to begin migration to Arbitrum". The prompt will only appear if they're wallet is connected to rinkeby

**Review by adamsoffer — CHANGES_REQUESTED** [review]

File: docs/video-miners/how-to-guides/l2-migration.md [hunk 5]
```diff
@@ -41,29 +41,26 @@ Other options are: [set up an Arbitrum node](https://developer.offchainlabs.com/
...
-    
+    If you prefer to sign using the wallet that you have connected to the explorer, click "Approve Migration" and approve the transaction using your browser extension
     <img src="/docs-assets/video-miners/how-to-guides/sign-web.png" alt="sign web" width="300"/>
-
     
     3b. **Sign using CLI**
     
+    Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amount will not appear until after you enter your Ethereum account address.
+
     If you'd rather sign a typed data message through the CLI, connect your wallet to the Explorer with any other account. You'll be asked to enter the public address of the orchestrator you wish to migrate.
```
Comment (adamsoffer) [inline docs/video-miners/how-to-guides/l2-migration.md:60]: Let's change the order of these two paragraphs so it reads:

"If you'd rather sign a typed data message through the CLI, connect your wallet to the Explorer with any other account. You'll be asked to enter the public address of the orchestrator you wish to migrate. Please note that if you are signing with the CLI and your connected wallet is NOT your orchestrator wallet, the stake amount will not appear until after you enter your Ethereum account address."

**Review by adamsoffer — CHANGES_REQUESTED** [review]
@hthillman last comment I swear :)

**Pull request merged**